src/assets/sharedTestStrings.js

```diff
@@ -7,3 +7,13 @@ export const responsiveWebDesignStack = '1. You can use HTML, JavaScript, ' +
   'Other projects will give you a chance to work with different technology ' +
   'stacks like React. We will accept and try to fix all issue reports that ' +
   'use the suggested technology stack for this project. Happy coding!';
+
+export const frontEndLibrariesStack = '1. You can use any mix of HTML, ' +
+  'JavaScript, CSS, Bootstrap, SASS, React, Redux, and jQuery to complete ' +
+  'this project. You should use a frontend framework (like React for ' +
+  'example) because this section is about learning frontend frameworks. ' +
+  'Additional technoligies not listed above are not recommended and using ' +
+  'them is at your own risk. We are looking at supporting other frontend ' +
+  'frameworks like Angular and Vue, but they are not currently supported. ' +
```

Comment:
I'm pretty sure the tests work as-is with Angular projects, but this warning can't hurt.

Reply:
As soon as we have example Angular projects to test against, we'll remove this.